docs/UserGuide.md

```diff
@@ -199,6 +197,24 @@ Examples:
 * `deletem 3` deletes the 3rd meeting in your currently displayed meeting list.
 * `deletem -1` returns an error for invalid input as -1 is not a positive integer.
 
+
+### Editing a meeting : `editm`
+
+Edits an existing meeting in your currently displayed meeting list.
+
+Format: `editm MEETING_INDEX [n/NAME] [d/DATE] [st/START_TIME] [et/END_TIME] [pt/PARTICIPANTS_INDEX]... [t/TAGS]...`
+
+* Edits the meeting at the specified `MEETING_INDEX`.
+* `MEETING_INDEX` refers to the index number shown in your currently displayed meeting list.
+* `MEETING_INDEX` **must be a positive integer** 1, 2, 3, …​
+* At least one of the optional fields must be provided.
+* Existing meeting details will be updated according to your input fields.
+
```

Comment:
Would it be good to write that any participants / tags specified in the input will replace all existing participants / tags?